api/v1alpha1/joinedcluster_types.go

```diff
@@ -96,10 +96,16 @@ type JoinedClusterConditions struct {
 type ClusterAgentInfo struct {
 	// Version of the cluster agent running in the spoke cluster.
 	Version string `json:"version"`
-	// Image of the cluster agent running int he spoke cluster.
+	// Image of the cluster agent running in the spoke cluster.
 	Image string `json:"image"`
 	// Last update time written by cluster agent.
 	LastUpdateTime metav1.Time `json:"lastUpdateTime"`
+	// ClusterName provides the infrastructure name of the spoke cluster
+	ClusterName string `json:"clusterName"`
```

Comment:
https://github.com/font/onprem/pull/11/commits/91c0fe69189f7cff13e334ac1c075e7f4a1e5025#diff-0e29d47123ec3bad423f6d376f53eae4R22
Can someone confirm if the above link presents the same `ClusterName` information that's listing here in the `ClusterAgentInfo` struct?

Reply:
Yeah it would. But here it is part of the reported ClusterAgentInfo. In the `RegisteredCluster` API it is part of the spec. That's what I was referring to - we should move it to the status.